### ISSUE TITLE
No audio streams for specific video

### ISSUE THREAD

**Issue by r0hin** [gatecrasher777/ytcog#35]
Hi,

With [this](https://youtu.be/qxiHjf5KGtw) song, no audioStreams seem to be available for some reason. This is despite being generated with a cookie to bypass age restriction. I seem to be logged in with `session.loggedIn` being `true`. 

Thought this was some behaviour that you might find interesting! Anyone able to reproduce it?

**Comment by gatecrasher777** [comment]
Wasn't able to replicate the issue this above song.  With logged in session I got:

```javascript
Available media streams:
{
  video: {
    '0': '1080p 921kbs mp4 h264 (21.0mb)',
    '1': '1080p 798kbs webm vp9 (21.9mb)',
    '2': '720p 452kbs webm vp9 (12.2mb)',
    '3': '720p 418kbs mp4 h264 (9.63mb)',
    '4': '480p 203kbs webm vp9 (5.48mb)',
    '5': '480p 175kbs mp4 h264 (4.08mb)',
    '6': '360p 234kbs mp4 h264 (10.8mb) with audio',
    '7': '360p 104kbs webm vp9 (3.22mb)',
    '8': '360p 81.5kbs mp4 h264 (1.98mb)',
    '9': '240p 113kbs mp4 h264 (3.16mb)',
    '10': '240p 56.3kbs webm vp9 (1.78mb)',
    '11': '144p 46.8kbs mp4 h264 (1.33mb)',
    '12': '144p 28.4kbs webm vp9 (895kb)'
  },
  audio: {
    '0': 'MEDIUM 164kbs webm opus (6.26mb)',
    '1': 'MEDIUM 131kbs mp4 aac (5.97mb)',
    '2': 'LOW 80.6kbs webm opus (3.13mb)',
    '3': 'LOW 61.3kbs webm opus (2.36mb)',
    '4': 'LOW 234kbs mp4 aac (10.8mb) with video'
  }
}
```
using examples/video_test.

**Comment by r0hin** [comment]
I ran `examples/video_test` from USA VPN with logged-in cookie and video ID `qxiHjf5KGtw` to get `Video status: ERROR (Sign in to confirm your age)`. 

I did get it working by using a different account cookie. I think it relates back to an older issue where specific account cookies don't seem to work despite `true` being returned from `session.loggedIn`. 

Anyhow, it is resolved now. Thanks for your help!
